core/frontend/src/render/webgl/glsl/Clipping.ts

```diff
@@ -124,7 +159,17 @@ export function addClipping(prog: ProgramBuilder) {
     });
   });
 
-  addModelViewMatrix(vert);
+  prog.frag.addUniform("u_colorizeIntersection", VariableType.Boolean, (program) => {
+    program.addProgramUniform("u_colorizeIntersection", (uniform, params) => {
+      uniform.setUniform1i(params.target.uniforms.branch.clipStack.colorizeIntersection ? 1 : 0);
+    });
+  });
+
+  prog.addUniform("u_clipIntersection", VariableType.Vec4, (program) => {
```

Comment:
This should probably be prog.frag.addUniform so that the u_clipIntersection uniform is not added to the vertex shader where it isn't used.  It most likely gets compiled out when the shaders compile, but we might as well not put it in there if it doesn't need to be.

Reply:
Can this be resolved?

Reply:
It's been addressed.